hr_office/i18n/sv_SE.po

```diff
@@ -82,7 +82,7 @@ msgstr "Senast uppdaterad"
 #. module: hr_office
 #: model:ir.model.fields,field_description:hr_office.field_hr_location_accessibility__location_id
 msgid "Location"
-msgstr "Plats"
+msgstr "Kontaktväg "
```

Comment:
Is really "Kontaktväg" the correct translation of "Location"?

Reply:
Nej, snarare Plats eller Platskod men kolla upp detta med någon som är säker. Jag ser ju inte sammanhanget heller. Handlar det om vilket lokalkontor som man ska infinna sig på så kanske det är bättre med Kontor. Detta med kontor respektive platskod vet Peter Ojaäär mycket om.

Reply:
Hej ,

Vi har diskuterat och kommit fram till följande. Jonas, Åsa, Henrik, Gunilla och föreslår att vi använder ordet Lokalkontor.

Not. På sikt i senare release kommer övergång att ske till ett sk. verksamhets ID
Istället. Men inte just nu. Den lägger vi i backloggen.

Mvh
Peter